CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Changes
 - Moved modules from `neptune.new` to `neptune` with compatibility imports and marked `neptune.new` as deprecated ([#1213](https://github.com/neptune-ai/neptune-client/pull/1213))
 - Removed `neptune.*` legacy modules ([#1206](https://github.com/neptune-ai/neptune-client/pull/1206))
+- Deprecated `get_project` function ([#1214](https://github.com/neptune-ai/neptune-client/pull/1214))
```

Comment:
Correct me if I'm wrong, but it's deprecated already and actually removed with 1.0, no?

Reply:
Yep